File: python/Borze.py
def main():
    s = input().strip()
    n = len(s)
    res = ""
    
    i = 0
    while i < n:
        if s[i] == '.':
            res += '0'
        elif s[i] == '-' and i+1 < n and s[i+1] == '.':
            res += '1'
            i += 1
        elif s[i] == '-' and i+1 < n and s[i+1] == '-':
            res += '2'
            i += 1
        i += 1
    print(res)
    
if __name__ == "__main__":
    main()

File: python/Borze.cpp
#include<iostream>
#include<vector>
#include<string>
using namespace std;

int main(){
    string str;
    cin >> str;
    int n = str.size();
    string res = "";

    for(int i = 0; i < n; i++){
        if(str[i] == '.'){
            res += '0';
        }else if(str[i] == '-' && str[i+1] == '.'){
            res += '1';
            i++;
        }else if(str[i] == '-' && str[i+1] == '-'){
            res += '2';
            i++;
        }
    }
    cout << res;
    return 0;
}